Makefile

```
@@ -192,6 +184,11 @@ kind-tests: kind-delete-cluster kind-bootstrap-cluster-dev kind-deploy-controlle
 
 .PHONY: install-crds
 install-crds:
+	@echo installing olm
+	# operator-sdk olm install
+	curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.26.0/install.sh -o install.sh
+	chmod +x install.sh
+	./install.sh v0.26.0
```

```suggestion
	curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.26.0/install.sh -o $(LOCAL_BIN)/install.sh
	chmod +x $(LOCAL_BIN)/install.sh
	$(LOCAL_BIN)/install.sh v0.26.0
```

I think most stuff like this already goes into `$(LOCAL_BIN)`, and it prevents them from being committed to the repo.

---

I don't see the change for this in the current commit.